larky/src/main/java/com/verygood/security/larky/modules/types/structs/SimpleStruct.java

```diff
@@ -116,4 +104,20 @@ public void debugPrint(Printer p) {
     p.append(")");
   }
 
+  protected Dict.Builder<String, Object> composeAndFillDunderDictBuilder() throws EvalException {
```

Comment:
I hate this name, but it's fine for us to change later. We should put a comment to say:
```

/**
  * Avoid un-necessary allocation if we need to override the immutability of the `__dict__` in a subclass for the caller.
  * */
```

Reply:
Done